### PR TITLE
README: added dependencies

### PR DESCRIPTION
**Pull request by Bowero**
I added the dependencies required to make binary file. Without these, the `build.sh` will fail.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,32 @@
 # Compiler for nop
+
 Please refer to the contribution guidelines [here](CONTRIBUTING.md)
 
-## Installation :
+## Installation
+
+### Dependencies
+
+Refer to [Installation of `antlr4` for this](https://github.com/Bowero/nopc-bootstrap#installation-of-antlr4)
```
Comment (Henrixounez) [inline README.md]: Please change the link to the official noplang repo

Comment (Henrixounez) [inline README.md]: Please change the link to the official noplang repo

Comment (lutherwenxu) [reply README.md]: Oh wow. Definitely did not catch that. 2 reviews is a good idea.

**Comment by lutherwenxu** [comment]
Rebase/squash to one commit instead of having two.

**Comment by lutherwenxu** [comment]
@Bowero You did not upload your key to GitHub so the commit is currently showing up as unverified. Please add it.

**Pull request merged**